Lib.AspNetCore.ServerTiming/Processors/DevelopmentEnvironmentBasedProcessor.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Lib.AspNetCore.ServerTiming.Http.Headers;
+using Microsoft.AspNetCore.Http;
+using System.Collections.Generic;
+
+namespace Lib.AspNetCore.ServerTiming.Processors
+{
+    /// <summary>
+    /// Base class for processors whose behaviour changes in the development environment
+    /// </summary>
+    public abstract class DevelopmentEnvironmentBasedProcessor : IServerTimingProcessor
+    {
+        /// <summary>
+        /// True if running in the development environment
+        /// </summary>
+        protected readonly bool IsDevelopment = false;
+
+
+#if !NETCOREAPP2_1 && !NET461
+        /// <summary>
+        /// Create an DevelopmentEnvironmentBasedProcessor
+        /// </summary>
+        /// <param name="hostEnvironment">The host environment used to determine if this is a development environment</param>
+        public DevelopmentEnvironmentBasedProcessor(Microsoft.Extensions.Hosting.IHostEnvironment hostEnvironment)
+        {
+            IsDevelopment = hostEnvironment.EnvironmentName == Microsoft.Extensions.Hosting.Environments.Development;
+        }
+#else
+        /// <summary>
+        /// Create an AllowInDevelopmentEnvironmentProcessor
```

Comment:
I believe the summary is outdated. In fact I would suggest using `<see cref=""/>` where appropriate. It allows easier navigation through generated API docs, and prevents names mismatch as well.